### PR TITLE
Update versioning for iOS and Android binary

### PR DESCRIPTION
**Pull request by IsaacHardy**
This gives us the ability to update the project files for android and ios by the `package.json` file.

## Creator

- [x] Build relevant tests if any apply
- [x] Ensure there are no new warnings (in app AND in storybook)
- [x] Upload an animated GIF showcasing the solution (if it applies)
- [x] Set a relevant reviewer
- [x] Add a line to the changelog documenting the high-level change this PR introduces

## Reviewer

- [ ] Review CI build results
- [ ] Review function and form on iOS
- [ ] Review function and form on Android
- [ ] Review new stories if any apply
- [ ] Review test coverage: Are all new or changed states tested?
- [ ] Review new test logic
- [ ] Review the changelog entry

### CODE REVIEW

File: mobile/package.json [hunk 1]
```diff
@@ -30,30 +30,34 @@
...
     "android": "react-native run-android",
+    "codecov": "cat ./coverage/lcov.info | ./node_modules/.bin/codecov",
+    "fixlint": "./node_modules/.bin/eslint . --fix",
     "ios": "react-native run-ios --simulator=\"iPhone X\"",
-    "test": "bash ../scripts/test-script.sh",
     "lint": "./node_modules/.bin/eslint . --ext .js",
-    "fixlint": "./node_modules/.bin/eslint . --fix",
-    "storybook": "rnstl --searchDir ./storybook ./src --pattern \"**/*.stories.js\" && storybook start -p 7007",
-    "codecov": "cat ./coverage/lcov.info | ./node_modules/.bin/codecov"
+<<<<<<< HEAD
```
Comment (conrad-vanl) [inline mobile/package.json]: Fix yo merge conflicts @IsaacHardy

Comment (IsaacHardy) [reply mobile/package.json]: And this is why you don't rush a PR because you have to poop.

Comment (conrad-vanl) [reply mobile/package.json]: 💩

**Comment by IsaacHardy** [comment]
@conrad-vanl I'd like to see what https://appcenter.ms/orgs/Apollos-Church/apps/Apollos-Church/build/branches/versioning/builds/33 does before merging.

**Comment by IsaacHardy** [comment]
@conrad-vanl Everything looks good! Got the 0.1.0 build on itunesconnect.

**Pull request merged**